source/set_up_3dsecure/index.html.md.erb

```diff
@@ -1,56 +1,71 @@
 ---
-title: Set up 3D Secure
+title: Set up Strong Customer Authentication
```

Comment:
where does this sit in the overall navigation? will it be easy to find for someone who is going live as well as those adding new 3ds flex credentials?

Reply:
It's a top level heading, at the same level as Go live. Perhaps we should put both SCA and 3d secure in the title?

Reply:
I think that we just need to make sure it works for these user groups:
- Worldpay users who already have a live Pay account, and now need to set up 3ds2.
- Worldpay users who don't yet have a live Pay account and are setting up their integration to go live. They should set up 3ds2 at the same time. 
- smartpay/epdq/stripe users who have a live account and need to check if they need to do anything for 3ds2 (they don't) (likely we'll send them an email with the update on how it's being handled, or maybe they are just speculatively checking the docs over the next few weeks when people start talking more about 3ds2?)
- smartpay/epdq/stripe users who don't yet have a live account and might want to know if there is anything they need to do to go live

Main thing is probably the Worldpay users as they've actually got to do something:
- Worldpay users with a live account will get specific comms saying 'set up 3ds flex'. This applies to about 80 or services (?) over the next few months and then this group's transition will be complete.
- Worldpay users that haven't gone live yet will get their 3ds flex credentials at the same time as their integration credentials. 

Will it be clear for the second group that as part of go live they need to add credentials? Should this sit somewhere in the go live flow?

Reply:
re. Gov Banking, the process is broadly the same. Worldpay will issue new credentials to the service in both cases. Gov Banking might provide some additional comms around cost and timing to their users; Worldpay will communicate directly with any direct users on this (i'm not sure of their process here, but not too worried about that).

Sounds good on the SCA front. And maybe once all existing users have been rolled over, we just keep the SCA section in go live and won't need a separate section in the docs. 

I'm not sure whether SCA is a term people are familiar with - they might be more familiar with 3DS / 3DS2. But technically, SCA is the requirement and the general regulatory area, and 3DS1/3DS2 are the implementation options.